include/deal.II/lac/solver_qmrs.h

```diff
@@ -65,7 +85,7 @@ DEAL_II_NAMESPACE_OPEN
  * to observe the progress of the iteration.
  *
  *
- * @author Guido Kanschat, 1999
+ * @author Guido Kanschat, Ingo Kligge 1999, 2017
```

Comment:
May you please write the author's list as 

> Guido Kanschat, 1999;  Ingo Kligge 2017

Although we're not completely consistent throughout the library, tend to extend the author's list in this way.